packages/chevrotain/src/parse/parser/traits/recoverable.ts

```diff
@@ -249,6 +253,10 @@ export class Recoverable {
     this: MixedInParser,
     expectedTokType: TokenType
   ): boolean {
+    if (!this.canTokenTypeBeDeletedInRecovery(expectedTokType)) {
```

Comment:
The project used to have 100%* code coverage.
The coverage inspection was disabled (probably by mistake) at some point so it is probably 99.5% now.

Anyhow, we are adding a logical branch here, so a test case should be added

Reply:
Sure, I will add a test.